apps/webapp/api/db/migrations/20250613213814_create_users_table.ts

```diff
@@ -0,0 +1,18 @@
+import { AbstractMigration, Info, ClientPostgreSQL } from "https://deno.land/x/nessie@2.0.11/mod.ts";
+
+export default class extends AbstractMigration<ClientPostgreSQL> {
+    async up(info: Info): Promise<void> {
+        await this.client.queryArray`CREATE TABLE users (
+            id UUID PRIMARY KEY,
+            email VARCHAR(255) NOT NULL,
+            display_name VARCHAR(255),
+            created_at TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
+            updated_at TIMESTAMP DEFAULT CURRENT_TIMESTAMP
+        )`;
+        await this.client.queryArray`CREATE UNIQUE INDEX users_email ON users (email)`;
```

Comment:
Also add an `updated_at` trigger. [Example](https://aviyadav231.medium.com/automatically-updating-a-timestamp-column-in-postgresql-using-triggers-98766e3b47a0)

Test this out. I don't normally do this explicitly because my framework handles it for me, so I'm unsure if this will work